src/goStatus.ts

```diff
@@ -10,8 +10,18 @@ import vscode = require('vscode');
 
 export let outputChannel = vscode.window.createOutputChannel('Go');
 
+export let statusBarItem = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Left);
+statusBarItem.command = 'go.show.outputPanel';
```

Comment:
We dont need to bind the status bar to a command. It doesnt provide any added benefits.